src/util/Util.js

```diff
@@ -578,6 +580,15 @@ class Util extends null {
    * @deprecated Use {@link BaseMessageOptions#allowedMentions} instead.
    */
   static removeMentions(str) {
+    if (!deprecationEmittedForRemoveMentions) {
+      process.emitWarning(
+        'This method is deprecated. Utilise allowed mentions when sending a message instead.',
```

Comment:
Why doesn't this follow the same pattern as the others?
```suggestion
        'This method is deprecated. Use MessageOptions#allowedMentions when sending a message instead.',
```